src/user.spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jest */
+
+import {
+    getUser,  
+    testConnection,
+    xo,
+  } from "./util";
+  
+  describe("user", () => {
+    let userId1, userId2;
+  
+    test("getAll() works", async () => {
```

Comment:
After thought, i think this test in not necessary since we will test the `getAll()` after.